modin/experimental/core/execution/native/implementations/hdk_on_native/expr.py

```diff
@@ -1042,8 +1105,8 @@ def __repr__(self):
         -------
         str
         """
-        if len(self.operands) == 1:
-            return f"({self.op} {self.operands[0]} [{self._dtype}])"
+        if pk := getattr(self, "partition_keys", None):
+            return f"({self.op} {self.operands} {pk} {self.order_keys} [{self._dtype}])"
```

Comment:
Is it not covered by tests?

Reply:
No, this is a __repr__ method of an internal class, mainly used for debugging.